packages/sources/tradermade-test/src/endpoint/live.ts

```diff
@@ -0,0 +1,30 @@
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { PriceEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { BatchEndpointTypes, buildBatchedRequestBody, constructEntry } from '../price-utils'
+
+const inputParameters = {
+  base: {
+    aliases: ['from', 'coin', 'symbol'],
+    required: true,
+    type: 'string',
+    description: 'The symbol of symbols of the currency to query',
+  },
+  quote: {
+    aliases: ['to', 'market', 'convert'],
```

Comment:
I think `market` should be an alias for `base` instead of `quote` based on the v2 adapter.

Reply:
Right, i also missed that point